source/clear-linux/guides/maintenance/time.rst

```diff
@@ -5,57 +5,58 @@ Set the time
 
 |CLOSIA| uses the `systemd-timesyncd.service` daemon to synchronize time.
 
-This section provides instructions to reset the time in your |CL| system when
+This guide describes how to reset the time in your |CL| system when
 the default :abbr:`NTP (Network Time Protocol)` servers cannot be reached.
 
-#. Install the `sysadmin-basic` bundle:
+#. Install the `sysadmin-basic` bundle.
 
-   .. code-block:: console
+   .. code-block:: bash
 
       swupd bundle-add sysadmin-basic
 
-#. Set your timezone. This example uses Los Angeles.
+#. Set your time zone. This example uses Los Angeles.
 
-   .. code-block:: console
+   .. code-block:: bash
 
       timedatectl set-timezone America/Los_Angeles
```

Comment:
Add "$ " to the front of the line

Reply:
Not done because it bars users from simple copy/paste.

Reply:
agreed!  We'll get this right eventually!